app/src/features/projects/components/ProjectUserForm.tsx

```diff
@@ -30,14 +30,15 @@ export const ProjectUserRoleYupSchema = yup.object().shape({
 });
 
 interface IProjectUserFormProps {
-  roles: ICode[];
+  roles: ICodeWithDescription[];
+  description?: string;
 }
 
 export const ProjectUserRoleFormInitialValues = {
   participants: []
 };
 
-const ProjectUserForm = (props: IProjectUserFormProps) => {
+const ProjectUserForm: React.FC<IProjectUserFormProps> = (props) => {
```

Comment:
(Functionally the same, just an older pattern thats being deprecated)

Instead of:
```
const ProjectUserForm: React.FC<IProjectUserFormProps> = (props) => {
```
Its better to say:
```
// without children
const ProjectUserForm = (props: IProjectUserFormProps) => {
// or with children
const ProjectUserForm = (props: PropsWithChildren<IProjectUserFormProps>) => {
```

`React.FC<myType>` is an old way of defining a react component (which includes `children`). In the vast majority of components, we don't leverage `children`, so most of the time that first fixed example is what we want.

Reply:
I resolved this, and a few others I found while fixing something else.